### PR TITLE
fix: original image with size

### PR DESCRIPTION
**Pull request by rgllm**
@riencoertjens this fixes the issue discussed here https://github.com/TylerBarnes/gatsby-wordpress-inline-images/issues/15.

Please let me know if you have any question.

### CODE REVIEW

File: src/gatsby-node.js [hunk 1]
```diff
@@ -107,14 +107,31 @@ const replaceImage = async ({
...
+  let imageNode;
 
-  const imageNode = await downloadMediaFile({
-    url,
-    cache,
-    store,
-    createNode,
-    createNodeId,
-  })
+  try{
```
Comment (TylerBarnes) [inline src/gatsby-node.js:113]: Thanks so much for this! It looks great. Would you mind adding a couple comments as to why we have to do this and what the second download is for?
I can also add them if you'd like, just don't want to mess with your PR :) Let me know

**Review by TylerBarnes — CHANGES_REQUESTED** [review]
Thanks so much for this! It looks great. Would you mind adding a couple comments as to why we have to do this and what the second download is for?
I can also add them if you'd like, just don't want to mess with your PR :) Let me know

**Comment by rgllm** [comment]
> Thanks so much for this! It looks great. Would you mind adding a couple comments as to why we have to do this and what the second download is for?
> I can also add them if you'd like, just don't want to mess with your PR :) Let me know

Done @TylerBarnes 👍

**Comment by rgllm** [comment]
@TylerBarnes any updates on this?

**Comment by TylerBarnes** [comment]
@rgllm apologies for the delay on this. I was on vacation and then was working super hard writing the new version of `gatsby-source-wordpress`! I joined the Gatsby core team recently. Heads up, when `gatsby-source-wordpress@v4` drops, this package will be deprecated. See https://github.com/gatsbyjs/gatsby/issues/19292 for more info

**Comment by TylerBarnes** [comment]
Also, this is published in `gatsby-wordpress-inline-images@1.1.0`! Thanks again!

**Comment by rgllm** [comment]
> @rgllm apologies for the delay on this. I was on vacation and then was working super hard writing the new version of `gatsby-source-wordpress`! I joined the Gatsby core team recently. Heads up, when `gatsby-source-wordpress@v4` drops, this package will be deprecated. See [gatsbyjs/gatsby#19292](https://github.com/gatsbyjs/gatsby/issues/19292) for more info

I've been following your work with v4 @TylerBarnes. Good job there, would love to talk with you about that 👍

**Comment by TylerBarnes** [comment]
@rgllm and I'd love to hear your thoughts! Feel free to dm me on twitter (@tylbar) or comment in the epic :) Hopefully soon I'll have an experimental release ready for people to try out

**Pull request merged**